static/src/javascripts/projects/commercial/modules/messenger/background.js

```diff
@@ -194,7 +232,18 @@ const setBackground = (specs: AdSpec, adSlot: Node): Promise<any> => {
         .then(({ backgroundParent, background }) =>
             updateStyles(backgroundParent, background)
         )
-        .then(({ backgroundParent }) => observer.observe(backgroundParent));
+        .then(({ backgroundParent, background }) => {
+            if (specs.scrollType === 'interscroller') {
+                renderStickyAdLabel(backgroundParent).then();
+                renderStickyScrollForMoreLabel(backgroundParent).then();
```

Comment:
Is it necessary to chain these empty `.then()` calls?

Reply:
Not really, but intelliJ warns about `Promise returned is ignored`. Also though that, it would be more precise that the method is returning Promise with the empty `then()`. I have now removed it